JASP-Engine/JASP/R/common.R

```diff
@@ -2635,7 +2637,8 @@ editImage <- function(plotName, type, height, width) {
 
 			# copy plot and check if we edit it
 			plot <- oldPlot
-			if (FALSE && type == "interactive" && isGgplot) {
+			if (type == "interactive" && isGgplot) {
+			#if (FALSE && type == "interactive" && isGgplot) {
```

Comment:
Could you change it back to `if (FALSE && type == "interactive" && ..`? We wouldn't want any users to accidentally try out this feature :)